content/news/006.open-tech-meeting-2024-05.de.md

```diff
@@ -0,0 +1,22 @@
+---
+published: 2024-04-13
+description: Vorstellen unseres Fortschrittes und Lokales Technisches Meetup
+title: Einladung zum Open Tech Meetup
+keywords: [ Tech, Update, Event ]
+authors: [ Maurice, Tassilo ]
+image: 006_open_tech_1.webp
+---
+
+Das Peeren soll bald losgehen! Deswegen wollen wir Euch zu unserem Open-Tech Meeting einladen.
```

Comment:
Der DD-IX ist Eure Peering-Plattform. In den letzten Monaten ist hinter den Kulissen viel passiert. In unserem Open Tech Meeting wollen wir davon berichten, Euer Feedback einsammeln und mit Euch zukünftige Pläne besprechen.

Reply:
"Der DD-IX ist Eure lokale Peering-Plattform"